includes/Admin/AnyCommentGenericSettings.php

```diff
@@ -425,74 +425,74 @@ public function init_settings () {
 				          ->number()
 				          ->set_id( self::OPTION_COMMENT_UPDATE_TIME )
 				          ->set_title( __( 'Comment Update Time', "anycomment" ) )
-				          ->set_description( esc_html( __( 'Number of minutes user can update his comment. "0" or empty for no limit.', "anycomment" ) ) ),
+				          ->set_description( esc_html( __( 'Number of minutes users can edit their comment. Use \"0\" or empty for no limit.', "anycomment" ) ) ),
 
 				     $this->field_builder()
 				          ->number()
 				          ->set_id( self::OPTION_COUNT_PER_PAGE )
-				          ->set_title( __( 'Number of Comments Loaded', "anycomment" ) )
-				          ->set_description( esc_html( __( 'Number of comments to load initially and per page.', "anycomment" ) ) ),
+				          ->set_title( __( 'Number of Comments to show on page load', "anycomment" ) )
```

Comment:
We have titles capitalized first letter, maybe we could make it the same way here, so that we won't need to fix it for many languages later

Reply:
```suggestion
				          ->set_title( __( 'Number of Comments to Show on Page Load', "anycomment" ) )
```